src/validation.cpp

```diff
@@ -329,11 +329,9 @@ void CChainState::MaybeUpdateMempoolForReorg(
     // been previously seen in a block.
     auto it = disconnectpool.queuedTx.get<insertion_order>().rbegin();
     while (it != disconnectpool.queuedTx.get<insertion_order>().rend()) {
+        auto accepted = AcceptToMemoryPool(*this, *it, GetTime(), /* bypass_limits= */true, /* test_accept= */false);
```

Comment:
This is a behaviour chage. With this change, ATMP() is always called, even if `fAddToMempol` is false or the transaction is a coinbase.